kore/src/Kore/Strategies/Verification.hs

```diff
@@ -212,13 +218,24 @@ verifyHelper breadthLimit searchOrder claims axioms (ToProve toProve) =
         -> (ReachabilityRule, Limit Natural)
         -> ExceptT Stuck simplifier [ReachabilityRule]
     verifyWorker provenClaims unprovenClaim@(claim, _) =
-        withExceptT wrapStuckPattern $ do
-            verifyClaim breadthLimit searchOrder claims axioms unprovenClaim
+        withExceptT (wrapStuckPattern . snd) $ do
```

Comment:
I think we can leave this as `wrapStuckPattern` and move `snd` into the implementation of `wrapStuckPattern`.